pom.xml

```diff
@@ -168,6 +168,41 @@
                 <version>0.4.1</version>
             </extension>
         </extensions>
+        <pluginManagement>
+        	<plugins>
```

Comment:
oops. looks like there are some tabs mixed in here where the rest of the file just uses spaces